deploy/roles/r-core/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+r_core_state: ""
```

Comment:
@lel99999 This should be `r_core_state: present` by default, to prevent R from updating unintentionally.